frappe_telegram/client.py

```diff
@@ -2,21 +2,40 @@
 
 import frappe
 from frappe_telegram.handlers.logging import log_outgoing_message
+from telegram.parsemode import ParseMode
 
 """
 The functions defined here is provided to invoke the bot
 without Updaters & Dispatchers. This is helpful for triggering
 bot interactions via Hooks / Controller methods
 """
 
+def send_message(message_text: str, parse_mode=None, user=None, telegram_user=None, from_bot=None):
+    '''
+    Send a message using a bot to a Telegram User
+
+    message_text: `str`
+        A text string between 0 and 4096 characters that will be the message
+    parse_mode: `str`
+        Choose styling for your message: `markdown`, `markdown2` or `html`. Default is `None`
+    '''
+
+    if parse_mode is not None:
+        if parse_mode == "markdown":
+            parse_mode = ParseMode.MARKDOWN
```

Comment:
@Abadulrehman won't it be better if we had the incoming argument `parse_mode` be of type Enum `ParseMode` itself ? Is there any other reason for it to be a `str` first which gets mapped to the actual enum ?

Reply:
No reason except convenience and managing imports. Instead of accessing the library's Enum, our wrapper can obfuscate all that.

It does open the door to make more mistakes though...

Reply:
Hm, we have been importing most of python-telegram-bots handlers in `frappe_telegram.__init__` module. Shall we import the same there ? So the end-developer could import from our wrapper, like all other handlers.

Reply:
Good idea. Made the changes. Have a look and let me know what you think.